test/Driver/Dependencies/mutual.swift

```diff
@@ -3,21 +3,26 @@
 // RUN: rm -rf %t && cp -r %S/Inputs/mutual/ %t
 // RUN: touch -t 201401240005 %t/*
 
-// RUN: cd %t && %swiftc_driver -c -driver-use-frontend-path %S/Inputs/update-dependencies.py -output-file-map %t/output.json -incremental -driver-always-rebuild-dependents ./main.swift ./other.swift -module-name main -j1 -v 2>&1 | %FileCheck -check-prefix=CHECK-FIRST %s
+// RUN: cd %t && %swiftc_driver -c -driver-use-frontend-path %S/Inputs/update-dependencies.py -output-file-map %t/output.json -incremental -driver-always-rebuild-dependents ./main.swift ./other.swift -module-name main -j1 -v -driver-show-incremental 2>&1 | %FileCheck -check-prefix=CHECK-FIRST %s
 
 // CHECK-FIRST-NOT: warning
 // CHECK-FIRST: Handled main.swift
 // CHECK-FIRST: Handled other.swift
+// CHECK-FIRST: Queuing main.swift (initial)
```

Comment:
I agree. I've moved the test to its own file. My plan is to, in future commits, add files like `test/Driver/Dependencies/driver-show-incremental-*.swift` for each interesting output in `-driver-show-incremental`. Does that sound good?